internal/common/startup.go

```diff
@@ -91,6 +93,19 @@ func readEnvironmentLogLevel() log.Level {
 	return log.InfoLevel
 }
 
+func readEnvironmentalReportCaller() bool {
```

Comment:
What's this?

Reply:
This adds the caller to the log.  This means you log out the file&line number in every log line.